README.md

```diff
@@ -1,5 +1,54 @@
 # sie-backend
 
-## Cloud Functions Tutorial
+## Cloud Functions Tutorial 
 
 https://cloud.google.com/functions/docs/first-python
+
+## Setup development environment
```

Comment:
Can you actually add some CLI for this section? For example, what is the command to install Google SDK?

Reply:
Sure! That is something I am working on right now (SIE-48).